webboot/webboot.go

```diff
@@ -1,10 +1,20 @@
 // Copyright 2019 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
-// Get the time the machine has been up
+//
 // Synopsis:
-//     webboot
+//     webboot [OPTIONS...] URL or name of ISO
```

Comment:
name of bookmark? just wondering.

Reply:
Yes, good point. Name of bookmark flows better with the description. Name of bookmark are names of OS/ISOs, but yes, I definitely agree with changing the comment to include name of bookmark instead. Thanks for pointing that out :)!